.github/workflows/actions.yml

```diff
@@ -126,7 +126,5 @@ jobs:
           pip install -r requirements.txt --progress-bar off --upgrade
           pip uninstall -y keras keras-nightly
           pip install -e "." --progress-bar off --upgrade
-      - name: Install pre-commit
-        run: pip install pre-commit && pre-commit install
```

Comment:
Why is this not needed?

Reply:
It comes in the requirements. and this is the right way we shouldn't manually list packages in our GitHub action flow. We keep forgetting to update these and we want them easily maintainable.